_posts/2015-05-23-photoacoustic-penetration.md

```diff
@@ -0,0 +1,24 @@
+
+PA imaging provides a guiding mechanism for medical procedures without the risks of ionising radiation and the cost of expensive equipment. Light from lasers is shone on photoabsorbers, such as haemoglobin; the resulting sound waves are detected by transducers and translated into visual images of the targeted region. PA also provides high-contrast images of the surgical tools being used in the medical procedure.
+
+However, this imaging technique has come with a severe limitation so far: the deeper the light goes into tissue, the less photo-energy per unit area can be delivered, thus reducing the resolution and constraining PA imaging’s penetration depth. Since the energy of the laser cannot be arbitrarily increased to probe even deeper into tissue &mdash; these matters being regulated by bodies such as the American National Standards Institute &mdash; a new approach has recently been explored. In laboratory (non-clinical) conditions, researchers modified their surgical apparatus so that a fiber-optic light source could be inserted along with it into the material under observation, allowing the light to be shone not from the surface but from within the material (i.e. it is irradiated *interstitially*). The transducers, responsible for picking up the sound waves, continued to be located externally. Tests were carried out on a laboratory surrogate for tissue known as a [tissue-mimicking phantom](http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3207386/) as well as prostate tissue from pigs and liver tissue from cows.
+
+The researchers found that the use of an interstitial light source improved the penetration depth of the imaging technique well beyond previously established limits. Wires inserted into the tissue surrogate at depths of 7, 20, 37 and 47 mm were clearly visible using the novel method, whereas the conventional method only showed the wires at the 7 and 20 mm. The tests on the prostate and liver tissue proved to be similarly successful. Shining light from within, the researchers demonstrated, was key to significantly expanding how deep within tissue PA imaging can be employed.
+
+Despite the successes of the laboratory studies and the fact that the size of the fiber-optic cables needed is compatible with the surgical instruments currently used, the outlook remains conservative. For example, the researchers noted that the energy absorbed per unit area in the region immediately surrounding the interstitial fiber-optic laser may be a concern, but suggested that this could be mitigated by increasing the diameter of the fiber end-cap or by adding a diffuser around its tip. 
+
```

Comment:
Something to sum it all up, e.g. potential future therapeutic uses, what is the next step (clinical trial?)